presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/glue/GlueHiveMetastore.java

```diff
@@ -179,15 +178,15 @@ private static AWSCredentialsProvider getAwsCredentialsProvider(GlueHiveMetastor
     {
         if (config.getAwsAccessKey().isPresent() && config.getAwsSecretKey().isPresent()) {
             return new AWSStaticCredentialsProvider(
-                new BasicAWSCredentials(config.getAwsAccessKey().get(), config.getAwsSecretKey().get()));
+                    new BasicAWSCredentials(config.getAwsAccessKey().get(), config.getAwsSecretKey().get()));
         }
         if (config.isUseInstanceCredentials()) {
             return InstanceProfileCredentialsProvider.getInstance();
         }
         if (config.getIamRole().isPresent()) {
             return new STSAssumeRoleSessionCredentialsProvider
-                .Builder(config.getIamRole().get(), "presto-session")
-                .build();
+                    .Builder(config.getIamRole().get(), "presto-session")
+                    .build();
```

Comment:
This probably should go to different commit, the one where you already fixing indentation.